src/test.php

```diff
@@ -155,19 +156,23 @@ function dropOnDropBaseHandler(e) {
 
     /* Evaluate question 7 */
     $evaluation7 = evaluateQuestion7();
-    $feedback7= $evaluation7["feedback"];
+    $feedback7 = $evaluation7["feedback"];
     $correctness7 = $evaluation7["fullPoints"];
     $result = $statement->execute(array('userId' => $_SESSION['userid'], 'solved_at' => date("Y/m/d"), 'questionId' => 6, 'correctness' => $correctness7));
 
     $evaluation8 = evaluateQuestion8();
-    $feedback8= $evaluation8["feedback"];
+    $feedback8 = $evaluation8["feedback"];
     $correctness8 = $evaluation8["fullPoints"];
     $result = $statement->execute(array('userId' => $_SESSION['userid'], 'solved_at' => date("Y/m/d"), 'questionId' => 7, 'correctness' => $correctness8));
 
+
     /* Evaluate Total */
     $evaluationTotal = evaluateTotal();
     $pointsTotal = $evaluationTotal["points"];
-    $feedbackTotal= $evaluationTotal["feedback"];
+    $feedbackTotal = $evaluationTotal["feedback"];
+
+    $statement2 = $pdo->prepare("INSERT INTO tests (userId, date, amount, correct) VALUES (:userId, :date, :amount, :correct) ON DUPLICATE KEY UPDATE amount = amount+1, correct = (correct+$correct)");
+    $result2 = $statement2->execute(array('userId' => $_SESSION['userid'], 'date' => date("Y/m/d"), 'amount' => 1, 'correct' => $pointsTotal));
```

Comment:
```suggestion
    $statement2 = $pdo->prepare("INSERT INTO tests (userId, date, amount, correct) VALUES (:userId, :date, :amount, :correct) ON DUPLICATE KEY UPDATE amount = amount+1, correct = (correct+$pointsTotal)");
```
Sorry, die correct variable hatte ich wohl vergessen zu entfernen bei meinem evaluation refactor